### ISSUE TITLE
Missing Config File (I Need Help Plss)

### ISSUE THREAD

**Issue by Kirisotu** [ldtteam/Piston-Unlimited#26]
I cant find the Config file in my Minecraft config folder for Multi-Piston, does anyone know how to solve this plss.
BTW it wont auto put the config file into my Minecraft config folder.(I tried Install and Uninstall didn't work.)

**Comment by MotionlessTrain** [comment]
It is in the world folder, in "serverconfigs"

**Comment by Kirisotu** [comment]
> It is in the world folder, in "serverconfigs"

So in order to change config file I need to buy my own server?

**Comment by Talyda** [comment]
Actually, is there even a config file for it?
I went looking, cos I assumed it would be in savefile/serverconfig but I can't find it at all

**Comment by MotionlessTrain** [comment]
> Actually, is there even a config file for it? I went looking, cos I assumed it would be in savefile/serverconfig but I can't find it at all

I haven't checked. But if there is one, it would be there

**Comment by Kirisotu** [comment]
> > Actually, is there even a config file for it? I went looking, cos I assumed it would be in savefile/serverconfig but I can't find it at all
> 
> I haven't checked. But if there is one, it would be there

It says at the bottom of the Multi-Piston Mod page (How can I increase the range?

- In the multi-piston.cfg file in your config folder.)
So I'm guessing that it has config folder because I really want to increase the range of Multi-Piston.
I tried Every possible thing to find the config file and I cant still find it.
I tried Removing all the configs in my Minecraft folder, Restart my whole pc, try downloading the same mod in diff device(laptop) still not their, and I tried uninstall and install.
